app/views/contexts/summary/question.py

```diff
@@ -55,13 +65,14 @@ def get_answer(self, answer_store, answer_id):
     def _build_answers(
         self,
         *,
-        answer_store,
-        question_schema,
-        block_id,
-        list_name,
-        return_to,
-        return_to_block_id,
-    ):
```

Comment:
If `Answers.serialize()` is changed to return a `dict` as per comment in`answer.py`, then this method can return `list[dict[str, Union[str, any]]]`